crates/proto/src/serialize/txt/rdata_parsers/dnskey.rs

```diff
@@ -0,0 +1,138 @@
+use crate::rr::dnssec::rdata::dnskey::DNSKEY;
+use crate::rr::dnssec::Algorithm;
+use crate::serialize::txt::{ParseError, ParseErrorKind, ParseResult};
+
+fn bit_is_set(value: u16, bit: usize) -> bool {
+    let mask = 1 << bit;
+    value & mask == mask
+}
+
+#[allow(deprecated)]
```

Comment:
Why do we need this?

Reply:
we no longer need it. it's a leftover from old version that mentioned `Algorithm::RSASHA1NSEC3SHA1`